MC-1417 Updating documentation for MC 5.1 (#113)
hazelcast/management-center-docker: README.md

By default, the container automatically sizes the Java heap memory suitable to the specified resource limit or 
available memory.

> ## Configuration options

Reviewer: This doc uses title case elsewhere. We should make sure that all headings are in the same case.

> With from Management Center version 5.1, configuration options have been consolidated. System properties and environment variables

Reviewer: ```suggestion
In Management Center version 5.1 and later, system properties and environment variables
```

> can be used interchangeably. The complete reference of available settings is available in the [Management Center Reference Manual](https://docs.hazelcast.com/management-center/5.1/deploy-manage/system-properties). Below you can find example configurations

Reviewer 2: `5.1` -> `latest` in the link
